### DISCUSSION TITLE
Disarm after throttle in idle

### DISCUSSION THREAD

**Discussion by Dawn-Hz** [iNavFlight/inav#10226] [Q&A] [answered]
I encountered an issue with iNav 6.1.0 where after I reduce the throttle to idle, the system disarms after about 3 seconds. To get the motor working again, I need to arm it again.

**Comment by MrD-RC** [comment]
Disarm or failsafe? If it's failsafe, check `failsafe_throttle_low_delay`. It should be set to 0 by default. But if this is higher than 0. It will failsafe after that period. 

https://github.com/iNavFlight/inav/blob/master/docs/Settings.md#failsafe_throttle_low_delay

If it is disarming. First, try updating to INAV 7.1.2. 6.1.0 is no longer supported. If it still happens. What is the `Disarmed by` on the stats?

**Reply by Dawn-Hz** [reply]
The problem was solved (disappeared), Thanks for the idea to look at the disarm message summary., but I encountered another issue: a pre-arm error stating "attitude calibration is not complete." I tried to recalibrate the accelerometer, but it failed. However, after I moved the VTX (AKK 3Watts), which was just a few millimeters away from the flight controller (Speedybee F405 wing), the accelerometer calibration passed.

**Comment by breadoven** [comment, marked as answer]
> I encountered an issue with iNav 6.1.0 where after I reduce the throttle to idle, the system disarms after about 3 seconds. To get the motor working again, I need to arm it again.

Sounds like it's just detecting it's landed in non autonomous modes (Acro, Poshold, Althold) and disarming when the throttle is low. Set `nav_disarm_on_landing `to OFF if you don't want it to do this, although it also won't disarm during an autonomous landing such as RTH in this case. If you want to allow it to disarm on landing but don't want it to disarm in non autonomous modes just don't fully lower the throttle after landing.

**Reply by Dawn-Hz** [reply]
Thanks for the idea. I will check it. I haven't performed a test flight yet because I haven't finished configuring the FC. I will not use autonomous landing. The platform is Avanti 80mm :-)

**Reply by Dawn-Hz** [reply]
![20240713_112059](https://github.com/user-attachments/assets/e09e4d61-2880-4f6c-a159-cd01f0bc112a)
So i need to write set nav_disarm_on_landing off ?

**Reply by breadoven** [reply]
Yes

**Reply by Dawn-Hz** [reply]
![Screenshot_20240713_134840_SpeedyBee](https://github.com/user-attachments/assets/a345bf71-1ec4-4516-8035-1c3e83f689d2)
It was set to "on" as you suspected.

**Reply by MrD-RC** [reply]
Not being funny. But it shouldn't disarm after lowering the throttle with an Avanti 80. That thing will still be going quickly after 3 seconds.
